src/towncrier/test/test_build.py

```diff
@@ -747,8 +748,8 @@ def do_build_once_with(version, fragment_file, fragment):
                     "--yes",
                 ],
             )
-            # not git repository, manually remove fragment file
-            Path(f"newsfragments/{fragment_file}").unlink()
+            # fragment files unknown to git are removed even without a git repo
+            # Path(f"newsfragments/{fragment_file}").unlink()
```

Comment:
I think that this needs a cleanup.

We should not merge commented code.

I think that we should repace this line with an assertion to check that the file is already removed.